vividus-to-azure-devops-exporter/src/main/java/org/vividus/azure/devops/facade/AzureDevOpsFacade.java

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright 2019-2021 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.vividus.azure.devops.facade;
+
+import static java.lang.String.format;
+
+import java.io.IOException;
+import java.util.List;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.vividus.azure.devops.client.AzureDevOpsClient;
+import org.vividus.azure.devops.configuration.AzureDevOpsExporterOptions;
+import org.vividus.azure.devops.converter.StepsToXmlConverter;
+import org.vividus.bdd.model.jbehave.Step;
+import org.vividus.util.json.JsonPathUtils;
+
+public class AzureDevOpsFacade
+{
+    private static final Logger LOGGER = LoggerFactory.getLogger(AzureDevOpsFacade.class);
+
+    private static final String API_VERSION = "?api-version=6.1-preview.3";
```

Comment:
is it always so?

Reply:
it's the latest version and required field, not sure that we should let end-users to configure this since models/endpoints might change